week1/vue_1/src/App.vue

```diff
@@ -0,0 +1,66 @@
+<template>
+  <h3>1주차 Vue 실습 - 할 일</h3>
+  <AddingInput @addToDo="addToDo"/>
+  <TodoCard v-for="todo,i in todoList" :key="i" v-bind:todo="todo" @editToDo="editToDo" @toggleToDo="toggleToDo" @deleteToDo="deleteToDo"/>
```

Comment:
```suggestion
  <TodoCard v-for="todo,i in todoList" :key="i" :todo="todo" @editToDo="editToDo" @toggleToDo="toggleToDo" @deleteToDo="deleteToDo"/>
```
바인드는 이렇게 생략할 수 있답니다!